File: Java/Nonmobile/Polymorphism/Pies.java
package com.example.javalib;

public class Pies extends Zwierze{

    public Pies(int waga) {
        super();
        System.out.println("Jestem psem, ważę " + waga + "kg");
    }

    @Override
    public void dajGlos() {
        System.out.println("hau hau");
    }

    @Override
    public void czymSieKarmie() {
        System.out.println("Karmię się: karmą i kośćmi");
    }
}

File: C#/Polymorphism/Polimorfizm/Pies.cs
namespace Polimorfizm;

public class Pies : Zwierze
{
    public Pies()
    {
        Console.WriteLine("Jestem psem");
    }
    
    override public void DajGlos()
    {
        Console.WriteLine("Głos psa - hau hau");
    }
    
    override public void Zjedz()
    {
        Console.WriteLine("Pokarm - mięso/karma dla psów");
    }

}